docs/source/troubleshooting.mdx

```diff
@@ -120,4 +120,28 @@ Another option is to get a better traceback from the GPU. Add the following envi
 >>> import os
 
 >>> os.environ["CUDA_LAUNCH_BLOCKING"] = "1"
-```
\ No newline at end of file
+```
+
+## Incorrect output when padding tokens aren't masked
+
+In some cases, you may notice the output `hidden_state` may be incorrect if the `input_ids` include padding tokens. For example, a batch of tensors may look like this before you pass it to `forward()`:
+
+```py
```

Comment:
Should we not load a model here so that the code can be executed? `model = BertModel.from_pretained("base-base-cased")`